src/shared/components/carousel/carousel-content/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { injectIntl } from 'react-intl';
+import { PropTypes } from 'prop-types';
+
+import Button from 'shared/components/button';
+import styles from './index.module.css';
+
+const CarouselContent = ({ icon, desc, image, intl: { messages } }) => (
```

Comment:
For consistency among other components, I think it would be better to call this component `CarouselItem`.

Reply:
Done.